packages/utils/src/translation-helpers.ts

```diff
@@ -67,8 +79,21 @@ export function applyTranslation(
     }
   });
 
-  traverse(blocks).forEach(function (el) {
-    // TODO: localized custom components inputs
+  traverse(blocks).forEach(function(el) {
+    const blockPath = [...this.path];
+    const blockIndex = blockPath.shift();
+    if (blockIndex) {
+      const parsedBlockIndex = parseInt(blockIndex);
+      const blockId = blocks![parsedBlockIndex].id;
+      const path = blockPath.join('#');
+      if (translation[`blocks.${blockId}#${path}`]) {
+        this.update({
+          ...el,
+          [locale]: translation[`blocks.${blockId}#${path}`].value,
```

Comment:
This should update the bindings not the el object

Reply:
check this line for reference https://github.com/BuilderIO/builder/blob/main/packages/utils/src/translation-helpers.ts#L87